doc/user/content/release-notes.md

```diff
@@ -117,6 +117,8 @@ for SQL statements.
 - Ensure `pg_get_constraintdef` [compatibility function] parses, though it
   remains unimplemented.
 
+- Support `generate_subscripts` for 1-dimensional `arrays`
```

Comment:
It works for n-dimensional arrays, no?

Reply:
no. We can't make those. You will get the following exception:
```
parsing multi-dimensional arrays is not supported
```

Reply:
Parsing them is unsupported, but constructing them via `ARRAY` literals is not.

Reply:
I wasn't aware of that.
```

materialize=> select generate_subscripts(ARRAY[ARRAY[1,2,3], ARRAY[5,6,7]], 1);
 generate_subscripts 
---------------------
                   1
                   2
(2 rows)

materialize=> select generate_subscripts(ARRAY[ARRAY[1,2,3], ARRAY[5,6,7]], 2);
 generate_subscripts 
---------------------
                   1
                   2
                   3
(3 rows)
```

Let me fix this in a quick follow-up.